package.json

```diff
@@ -34,13 +34,17 @@
 		"jsonschema": "^1.4.1",
 		"minimatch": "^9.0.4",
 		"mkdirp": "^3.0.1",
+		"repo-report": "^0.3.0",
 		"yargs": "^17.7.2"
 	},
 	"devDependencies": {
-		"@ljharb/eslint-config": "^21.1.1",
-		"auto-changelog": "^2.5.0",
+		"@eslint/js": "^9.31.0",
+		"@ljharb/eslint-config": "^21.1.0",
+		"aud": "^2.0.4",
+		"auto-changelog": "^2.4.0",
```

Comment:
once you delete `eslint.config.mjs`, you need to put this file back as well.

we do not want to update dependencies since your code doesn't require any new packages.